### DISCUSSION TITLE
Create entries from backend

### DISCUSSION THREAD

**Discussion by buskol** [umbraco/Umbraco.Forms.Issues#1509] [Ideas]
Umbraco forms are used for more than frontend form submissions - e.g our support uses the backend when talking to clients on the phone and often register them for events etc. I see more scenarios where it would be useful to be able to create an entry in the backend.

If the request is approved, I think a natural way to implement it, would be a button next to the 'Export' button in the Entries view opening an infinite editing window.

~~BTW, is it possible to add other buttons next to 'Export´?~~ _Solved it with $httpProvider.interceptors..._

TIA :-)

**Comment by TvGessel** [comment]
For this requirement, we recommend creating a separate page or a custom backoffice component as a workaround to achieve the desired layout. This won't be implemented for now.
